src/main/java/manga/Manga.java

```diff
@@ -1,15 +1,14 @@
 package manga;
 
 import author.Author;
-import com.google.gson.annotations.Expose;
+import storage.ExcludeInSerializationAnnotation;
 
 import java.util.Objects;
 
 public class Manga {
-    @Expose
     private String mangaName;
+    @ExcludeInSerializationAnnotation
```

Comment:
This will prevent the circular reference/building of the json from `Manga` to `Author` and back to `Manga`. Great work!